src/osm_configurator/view/states/main_window.py

```diff
@@ -86,7 +84,7 @@ def __init__(self, export_controller: IExportController, category_controller: IC
                      main_window_constants_i.MainWindowConstants.MAIN_WINDOW_HEIGHT_MINIMUM.value)
         self.resizable(True, True)
 
-        # Configurating the grid for the Application
+        # Configuration the grid for the Application
```

Comment:
maybe write Configuration of the grid ...
Otherwise your fix just makes the sentence looks weird

Reply:
fixed